### PR TITLE
Use SingularityCE and -ce package name consistently

### LINKED ISSUE #5: Apply consistent use of SingularityCE / singularity-ce through the docs
Ensure the CE suffix is used consistently so there is clear differentiation.

### PR DESCRIPTION
**Pull request by dtrudg**
## Description of the Pull Request (PR):

Use a replacement for the Singularity noun, so SingularityCE docs more clearly differentiate from older Singularity versions. Will also allows Sylabs SingularityPRO docs to use SingularityPRO as the noun, which is beneficial as the long term supported PRO versions can drift in content from the CE release due to backport through time.

## This fixes or addresses the following GitHub issues:

- Fixes #5

### CODE REVIEW

File: environment_and_metadata.rst [hunk 1]
```diff
@@ -749,12 +749,12 @@ the SIF file metadata descriptor.
     ├── libs
     ├── runscript
     ├── runscript.help
-    ├── Singularity
+    ├── {Singularity}
```
Comment (wobito) [inline environment_and_metadata.rst:752]: No sure if this changes... just built a container and checked.

```sh
➜  .singularity.d sn --version
singularity-ce version 3.8.0-rc.2
➜  .singularity.d pwd
/tmp/debian/.singularity.d
➜  .singularity.d ll
total 28K
drwxr-xr-x 2 wobito wobito 4.0K May 20 12:37 actions
drwxr-xr-x 2 wobito wobito 4.0K May 20 12:37 env
-rw-r--r-- 1 wobito wobito  352 May 20 12:37 labels.json
drwxr-xr-x 2 wobito wobito 4.0K May 20 12:37 libs
-rwxr-xr-x 1 wobito wobito 1.1K May 20 12:37 runscript
-rw-r--r-- 1 wobito wobito   39 May 20 12:37 Singularity
-rwxr-xr-x 1 wobito wobito   10 May 20 12:37 startscript
➜  .singularity.d cat Singularity
bootstrap: docker
from: debian:latest
```

**Review by wobito — COMMENTED** [review]
I think I found just the one ref that shouldn't be

File: environment_and_metadata.rst (hunk 1 above)
Comment (dtrudg) [reply environment_and_metadata.rst:752]: 👍 good spot, thanks. I'll take care of this!

Comment (dtrudg) [reply environment_and_metadata.rst:752]: The file tree is still appropriate for a container built out with all the possible def file sections.

**Review by wobito — APPROVED** [review]
LGTM

**Pull request merged**